bool cmd # Takeoff=True  Land=False
float32 altitude
---